packages/plugins/prometheus/test/prom.spec.ts

```diff
@@ -494,6 +514,72 @@ describe('Prom Metrics plugin', () => {
       expect(await metricCount('graphql_envelop_execute_resolver', 'count')).toBe(2);
       expect(await metricCount('graphql_envelop_deprecated_field')).toBe(1);
     });
+
+    it.skip('Should track deprecated fields in mutation input', async () => {
```

Comment:
Can you remove this test case if the implementation is not covered?

Reply:
This could be tracked with the https://www.envelop.dev/plugins/use-extended-validation plugin 🤔

Reply:
Thanks, I'll remove the skipped test. And I'll have a look if `use-extended-validation` could work in our case 👀